### ISSUE TITLE
Is it possible to change mode and version?

### ISSUE THREAD

**Issue by epasquali** [davidshimjs/qrcodejs#180]
The QR code that is being generated for around 128 characters for me is too "dense", which means that is very difficult for me to then scan with a VGA camera, even when scaled up a lot in print.

Is there a way to know and change the QR encoding version? I've changed the error correction to low, but clearly this is not a desired way to make the code simpler. Changing the encoding version while keeping error correction relatively high would be ideal. How can we do this? The readme file doesn't say....

**Comment by zonahaha** [comment]
Did you solve your problem? I want to change the mode too, but i dont know how to, I've tried a lot but doesn't work

**Comment by ArcadiaEngineering** [comment]
No, I ended up simply lowering the error correction to the lowest level. That seemed good enough for a VGA camera given the code I was using. Didn't find a way to change the encoding version to keep the error correction high. :(

**Comment by WebDevSwe** [comment]
It is possible to set the quality between L, M, Q and H as follows:
**High:**
correctLevel: QRCode.CorrectLevel.H
**Low:**
correctLevel: QRCode.CorrectLevel.L
